src/main/kotlin/lotto/domain/LottoTicket.kt

```diff
@@ -0,0 +1,30 @@
+package lotto.domain
+
+import lotto.domain.LottoNumber.Companion.MINIMUM_NUMBER
+import lotto.domain.LottoNumber.Companion.MAXIMUM_NUMBER
+
+class LottoTicket constructor(val lottos: List<Lotto>) {
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
class LottoTicket(val lottos: List<Lotto>) {
```